api/apps/api/src/modules/legacy-project-import/domain/events/halt-legacy-project-import.events.ts

```diff
@@ -0,0 +1,6 @@
+import { ResourceId } from '@marxan/cloning/domain';
+import { IEvent } from '@nestjs/cqrs';
+
+export class HaltLegacyProjectImport implements IEvent {
```

Comment:
Events should have past tense names, they represent something that happened. `HaltLegacyProjectImport` seems like a command name to me (something imperative)